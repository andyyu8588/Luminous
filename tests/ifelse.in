if (true) {
  xd = "XD";
  if (not(false)) {
    xd1 = xd + "XD1";
    print(xd + xd1);
  }
} else {
  print("THIS SHOULD NOT PRINT!");
}

if (false) {
  if (1 equals 1) {
    print("THIS SHOULD NOT PRINT!");
  }
  if (1 equals 2) {
    print("THIS SHOULD NOT PRINT!");
  }
} else {
  xd2 = "XD2";
  print(xd2);
}