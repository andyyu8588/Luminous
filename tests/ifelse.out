XDXDXD1
XD2
